lib/msal-browser/src/custom_auth/core/utils/ArgumentValidator.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { InvalidArgumentError } from "../error/InvalidArgumentError.js";
+
+export class ArgumentValidator {
+    static ensureArgumentIsNotEmptyString(
+        argName: string,
+        argValue: string | undefined,
+        correlationId?: string
+    ): void {
+        if (!argValue || argValue.trim() === "") {
+            throw new InvalidArgumentError(argName, correlationId);
+        }
+    }
+
+    static ensureArgumentIsNotNullOrUndefined<T>(
```

Comment:
Can you please replace static class functions with standalone ones? For example: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/53f7d511c4abe24e3182f35f70c87de2b6aa3c7a/lib/msal-browser/src/interaction_client/RedirectClient.ts#L53.

This facilitates bundle minification.